src/main/java/ejercicio/banco/repository/CsvAccountRepository.java

```diff
@@ -82,9 +101,15 @@ public void store(Account account) throws IOException {
     }
 
     @Override
-    public void update(int id, Account account) throws FileNotFoundException {
+    public void update(int id, Account account) {
 
-        List<String> in = reader.read(FILENAME);
+        List<String> in = null;
+        try {
+            in = reader.read(FILENAME);
+
+        } catch (FileNotFoundException e) {
+            in = null;
```

Comment:
This would throw a `NullPointerException` in the line 114 in case the `FILENAME` doesn't exist. If the file doesn't exist, what would you do it?